optuna_dashboard/_serializer.py

```diff
@@ -172,20 +170,21 @@ def serialize_study_detail(
 
 def serialize_preference_history(
     system_attrs: dict[str, Any],
-) -> list[History]:
-    histories: list[History] = []
+) -> list[dict[str, Any]]:
+    histories: list[dict[str, Any]] = []
     for k, v in system_attrs.items():
         if not k.startswith(_SYSTEM_ATTR_PREFIX_HISTORY):
             continue
         choice: dict[str, Any] = json.loads(v)
         if choice["mode"] == "ChooseWorst":
-            history: ChooseWorstHistory = {
+            history = {
                 "mode": "ChooseWorst",
                 "id": choice["id"],
                 "preference_id": choice["preference_id"],
                 "timestamp": choice["timestamp"],
                 "candidates": choice["candidates"],
                 "clicked": choice["clicked"],
+                "enabled": is_preference_valid(system_attrs, choice["preference_id"]),
```

Comment:
There are two terms `enabled` and `valid` here. How about making it more consistent like below?

```suggestion
                "is_removed": is_preferential_removed(system_attrs, choice["preference_id"]),
```